### PR TITLE
Don't add a default site to emails from SITE_ID on save

### PR DESCRIPTION
**Pull request by lynnco**

### CODE REVIEW

**Comment by lynnco** [comment]
@kelseyq up for review 👍

File: tests/notification/test_sites.py [hunk 1]
```diff
@@ -31,42 +31,54 @@ def __exit__(self, *args):
...
+
+        with TempSiteID(site_1.id):
+            self.assertEqual(EmailNotification.objects.on_site().count(), site_1_pages)
 
         with TempSiteID(site_2.id):
             self.assertEqual(EmailNotification.objects.on_site().count(), site_2_pages)
 
     @override_settings()
-    def test_site_not_added_multiple_times_on_save(self):
+    def test_site_overriden_on_save(self):
```
Comment (kelseyq) [inline tests/notification/test_sites.py]: this is replacing the original function of the test, no? shouldn't save still be called twice?

**Comment by kelseyq** [comment]
what happens if you attempt to save with no site added? can you add coverage of that?

File: tests/notification/test_sites.py (hunk 1 above)
Comment (lynnco) [inline tests/notification/test_sites.py]: we don't subclass save anymore, so I couldn't think of a reason to test double saves

Comment (kelseyq) [inline tests/notification/test_sites.py]: ok, fair enough!

Comment (lynnco) [inline tests/notification/test_sites.py]: (I wrote that test name wrong though)

**Comment by lynnco** [comment]
> what happens if you attempt to save with no site added? can you add coverage of that?

https://github.com/SexualHealthInnovations/callisto-core/pull/172/commits/085f172d26b91c5736fd83ef661d86140b70da43

**Pull request merged**